.github/workflows/tekton_bundle_push.yaml

```diff
@@ -7,6 +7,9 @@ on:  # yamllint disable-line rule:truthy
       - 'catalog/*/*/*/*.yaml' # E.G. catalog/pipeline/release/0.2/release.yaml
       - '!catalog/*/*/*/samples/**' # negated paths
       - '!catalog/*/*/*/tests/**'
+      - 'catalog/task/upstream/*/*/*/*.yaml' # E.G. catalog/task/upstream/tekton/kubernetes-actions/0.2/kubernetes-actions.yaml
```

Comment:
I was wondering if this was enough here. And it turns out it wasn't. For reference, here's the follow up PR that enabled also pushing tekton tasks in the `task/upstream` dir: https://github.com/redhat-appstudio/release-service-bundles/pull/132